src/app/qgsdiagramproperties.cpp

```diff
@@ -59,6 +59,9 @@ QgsExpressionContext QgsDiagramProperties::createExpressionContext() const
 
 QgsDiagramProperties::QgsDiagramProperties( QgsVectorLayer *layer, QWidget *parent, QgsMapCanvas *canvas )
   : QWidget( parent )
+  , mPlacePointBtnGrp( nullptr )
```

Comment:
Just set them in the header (`QButtonGroup mPlacePointBtnGrp = nullptr;`)

Reply:
Actually, that's done already. Kill em.

Reply:
Ok I will remove them. For the recall, I mostly follow the labeling logic exposed at https://github.com/qgis/QGIS/blob/07a570fd0560d1cb31a179b95e75cc5d5caf649b/src/gui/qgstextformatwidget.h#L120 and https://github.com/qgis/QGIS/blob/ce92ff61580c257096fc922c07926ea893372891/src/gui/qgstextformatwidget.cpp#L37. Should they be removed too?